projects/packages/connection/src/class-tokens.php

```diff
@@ -642,6 +642,10 @@ public function remove_lock() {
 
 	/**
 	 * Check if the domain is locked, remove the lock if needed.
+	 * Possible scenarios:
+	 * - lock expired, site URL matches the lock URL: remove the lock, return false.
+	 * - lock not expired, site URL matches the lock URL: return false.
+	 * - site URL does not match the lock URL (expiration date is ignored): return true, do not remove the lock.
```

Comment:
Yes, even if the lock is expired, it will still return `true` if the URL doesn't match.
The expiration date is used on the original site only.
If the URL's match, `is_locked` will return `false`.
But if the URL's match _and_ the lock is expired, it will return `false` and remove the lock.

We set it to one hour to make sure that export is completed when it's time to remove it from the original site.

Reply:
To summarize the discussion from Slack, we need the expiration date to see when the lock is ready to be removed.
Otherwise it will may there forever if we fail to remove it explicitly.